server/services/reportedAdjudicationsService.ts

```diff
@@ -298,6 +298,10 @@ export default class ReportedAdjudicationsService {
     const friendlyName =
       (prisonerResult && convertToTitleCase(`${prisonerResult.firstName} ${prisonerResult.lastName}`)) || ''
     const reportingOfficer = (reporterName && convertToTitleCase(reporterName)) || ''
+    const nextSheduledHearingDate =
+      reportedAdjudication.status === ReportedAdjudicationStatus.SCHEDULED
+        ? reportedAdjudication.hearings[reportedAdjudication.hearings.length - 1].dateTimeOfHearing
+        : ''
```

Comment:
From the screenshot it looks like Simon wants there to be a hyphen if there's no scheduled hearing date to display here. So I think the best way to do this with your approach is to return `null` here if the status isn't SCHEDULED and then below (I will leave a comment where) you can use `|| "-"` as otherwise it will return an invalid date